<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppointmentEditView</class>
 <widget class="QWidget" name="AppointmentEditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑就诊记录</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelPatientName">
       <property name="text">
        <string>病人姓名：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtPatientName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelDoctorName">
       <property name="text">
        <string>医生姓名：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtDoctorName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelAppointmentDate">
       <property name="text">
        <string>就诊日期：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelDiagnosis">
       <property name="text">
        <string>诊断结果：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtDiagnosis"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelMedication">
       <property name="text">
        <string>开具药品：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="txtMedication"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>